### DISCUSSION TITLE
Maintain embedded 608 / 708 captions with edits/ transcoding ?

### DISCUSSION THREAD

**Discussion by tvadi** [mifi/lossless-cut#1966] [General]
Hi, trying to find a replacement for VideoRedo since the owner passed away (RIP Dan).  I work in TV in the states and need to maintain 608 / 708 captions.  We get .MPG and .MXF files with embedded captions and I need to do minor edits like chopping off color bars or a slate at the beginning of the program, also need to upconvert at times to 1080i or 720p, final output is always .MPG (mpeg-2).  Can lossless-cut do the edits and up-conversion while maintaining the captions?  Thanks

**Comment by mifi** [comment]
hi. losslesscut does not do any upconversion because those are lossy operations that require re-encoding. see faq.
